app/controllers/agent/matching/ClientDetailsController.scala

```diff
@@ -56,6 +58,8 @@ class ClientDetailsController @Inject()(val auditingService: AuditingService,
 
   def show(isEditMode: Boolean): Action[AnyContent] = Authenticated.async { implicit request =>
     implicit user =>
+      val agentReferenceNumber: Option[String] = Some(user.arn)
+      startAgentSignupAudit(agentReferenceNumber)
```

Comment:
```suggestion
      startAgentSignupAudit(agentReferenceNumber = Some(user.arn)
```
Very slight simplification